app/src/main/res/layout/fragment_web.xml

```diff
@@ -14,39 +14,44 @@
   ~ See the License for the specific language governing permissions and
   ~ limitations under the License.
   -->
-
-<android.support.v4.widget.NestedScrollView
+<android.support.v4.widget.SwipeRefreshLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    app:layout_behavior="@string/appbar_scrolling_view_behavior"
-    android:id="@+id/nested_scroll_view"
-    android:background="?attr/colorCardBackground"
-    android:scrollbarStyle="insideOverlay"
-    android:scrollbars="vertical"
+    android:id="@+id/swipe_container"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content">
+    android:layout_height="match_parent"
```

Comment:
Can this be `wrap_content`? I remember last time I had to make it `wrap_content`, or else there are lots of empty spaces at the end of the page.